experiments/run_ooo.py

```diff
@@ -1,28 +1,29 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
 
-import utilities as u
+import run_utility as u
 
 aggregators = [
-                "bclassic2",
+                #"bclassic2",
                 "bclassic4",
                 "bclassic8",
-                "bfinger2",
+                #"bfinger2",
                 "bfinger4",
                 "bfinger8",
                 #"bfinger16",
                 #"bfinger32",
               ]
 
-degrees = [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 
-           1*u.KB, 2*u.KB, 4*u.KB, 8*u.KB, 16*u.KB, 32*u.KB, 64*u.KB, 128*u.KB, 256*u.KB, 512*u.KB,
-           1*u.MB, 2*u.MB, 4*u.MB]
-
+degrees = [1, 512, 64*u.KB, 2*u.MB]
+#degrees = [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 
+#           1*u.KB, 2*u.KB, 4*u.KB, 8*u.KB, 16*u.KB, 32*u.KB, 64*u.KB, 128*u.KB, 256*u.KB, 512*u.KB,
+#           1*u.MB, 2*u.MB, 4*u.MB]
+#
 base_window_sizes = [4*u.MB]
                     #[1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 
                     # 1*u.KB, 2*u.KB, 4*u.KB, 8*u.KB, 16*u.KB, 32*u.KB, 64*u.KB, 128*u.KB, 256*u.KB, 512*u.KB,
                     # 1*u.MB, 2*u.MB, 4*u.MB]
 
-base_iterations = 100 * u.MILLION
+base_iterations = 10 * u.MILLION
```

Comment:
This doesn't seem to match the number in the processing script. I'd guess keeping it at 100m will probably be better given how fast current machines are.

Reply:
Same comments as above, but we should have some values defined per experiment so that the run and process scripts use the same value.